src/net/sourceforge/kolmafia/request/RelayRequest.java

```diff
@@ -869,15 +876,15 @@ public boolean sendBreakPrismWarning(final String urlString) {
       // If you don't have any coins, nothing to spend
       if ( InventoryManager.getCount( ItemPool.COIN ) > 0 )
       {
-        if ( this.getFormField( CONFIRM_RALPH2 ) == null )
+        if ( this.getFormField( Confirm.CONFIRM_RALPH2 ) == null )
```

Comment:
Why is this not Confirm.RALPH2?

Reply:
it's commented-out code, so didn't get picked up by the refactor. I'll adjust it.